cmd/titus-vpc-service/main.go

```diff
@@ -370,10 +358,6 @@ func bindVariables(v *pkgviper.Viper) {
 		panic(err)
 	}
 
-	if err := v.BindEnv(sslTitusAgentCAFlagName, "SSL_TITUSAGENT_CA"); err != nil {
```

Comment:
How is the root cert that the client certs are signed with getting loaded into the grpc server?

Reply:
That comes from SSL_CA, as part of the `tls_config`.